Dockerfile

```diff
@@ -106,19 +105,6 @@ RUN cat /etc/issue
 # we follow instructions from
 # https://computational-science.mpsd.mpg.de/docs/mpsd-hpc.html#loading-a-toolchain-to-compile-octopus
 
-RUN mkdir -p build-octopus
-WORKDIR /home/user/build-octopus
-RUN git clone https://gitlab.com/octopus-code/octopus.git
-WORKDIR /home/user/build-octopus/octopus
-RUN pwd
-RUN ls -l
-RUN autoreconf -fi
-RUN mkdir _build
-WORKDIR /home/user/build-octopus/octopus/_build
-RUN pwd
-RUN cp /home/user/mpsd-software/${MPSD_RELEASE}/spack-environments/octopus/${TOOLCHAIN}-config.sh .
-RUN ls -l
-ADD install-octopus.sh .
-RUN bash install-octopus.sh ${TOOLCHAIN} ${MPSD_RELEASE}
+RUN bash install-octopus.sh ${TOOLCHAIN} ${MPSD_RELEASE} /home/user/octopus_build
```

Comment:
I would prefer to first have the RELEASE argument and then the toolchain, i.e.

`install-octopus.sh dev-23a foss2022a-mpi /home/user/octopus_build` instead of 
`install-octopus.sh foss2022a-mpi dev-23a  /home/user/octopus_build`.

Then the order is the same as we have for `mpsd-software`.